src/Pokemon.java

```diff
@@ -0,0 +1,121 @@
+package src;
+
+public abstract class Pokemon {
+    private double healthPoint;
+    private double experiencePoints;
+
+    private String name;
+    private int level;
+    private double height;
+    private double weight;
+    private double defence;
+    private double attack;
+    private double accuracy;
+    private String special;
+
+
+    public abstract void speaks();
+    public abstract void eats(String food);
+    public abstract void attack();
+
+    public void getStats(){
+        System.out.println("Level: " + level);
+        System.out.println("Name: " + name);
+        System.out.println("Health Point: " + healthPoint);
+        System.out.println("Experience Points: " + experiencePoints);
+        System.out.println("Height: " + height);
+        System.out.println("Weight: " + weight);
+        System.out.println("Defence: " + defence);
+        System.out.println("Attack: " + attack);
+        System.out.println("Accuracy: " + accuracy);
+        System.out.println("Special: " + special);
+    }
+    public void setExperiencePoints(double experiencePoints) {
+        this.experiencePoints = experiencePoints;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getLevel() {
+        return level;
+    }
+
+    public void setLevel(int level) {
+        this.level = level;
+    }
+
+    public double getHeight() {
+        return height;
+    }
+
+    public void setHeight(double height) {
+        this.height = height;
+    }
+
+    public double getWeight() {
+        return weight;
+    }
+
+    public void setWeight(double weight) {
+        this.weight = weight;
+    }
+
+    public double getDefence() {
+        return defence;
+    }
+
+    public void setDefence(double defence) {
+        this.defence = defence;
+    }
+
+    public double getAttack() {
+        return attack;
+    }
+
+    public void setAttack(double attack) {
+        this.attack = attack;
+    }
+
+    public double getAccuracy() {
+        return accuracy;
+    }
+
+    public void setAccuracy(double accuracy) {
+        this.accuracy = accuracy;
+    }
+
+    public String getSpecial() {
+        return special;
+    }
+
+    public void setSpecial(String special) {
+        this.special = special;
+    }
+
+    public Pokemon(double healthPoint, double experiencePoints) {
```

Comment:
Deze constructor staat verstopt tussen de getters en setters. Beter is om een vaste volgorde aan te houden: 
- constanten
- variabelen
- constructors
- getters/setters
- andere methodes
Zo weet je altijd dat je constructors bovenaan staan en hoef je niet lang te zoeken